packages/litexa-html/lib/htmlHandler.js

```diff
@@ -1,23 +1,57 @@
 module.exports = (context) => {
-    const contextHasHTML = () => {
-        const interfaces =
-            context &&
-            context.event &&
-            context.event.context &&
-            context.event.context.System &&
-            context.event.context.System.device &&
-            context.event.context.System.device.supportedInterfaces;
-        return interfaces && interfaces.hasOwnProperty('Alexa.Presentation.HTML');
-    };
+  const contextHasHTML = () => {
+    const interfaces =
+      context &&
+      context.event &&
+      context.event.context &&
+      context.event.context.System &&
+      context.event.context.System.device &&
+      context.event.context.System.device.supportedInterfaces;
+    return interfaces && interfaces.hasOwnProperty('Alexa.Presentation.HTML');
```

Comment:
Would consider adding a minimum version check here, since we're not sure how long `0.2` will stick around. Maybe something like:
```
const interfaces =
    context &&
    context.event &&
    context.event.context &&
    context.event.context.System &&
    context.event.context.System.device &&
    context.event.context.System.device.supportedInterfaces;

const htmlInterface = interfaces && interfaces['Alexa.Presentation.HTML'];

return htmlInterface &&
    (htmlInterface.runtime == null || parseFloat(htmlInterface.runtime.maxVersion) >= 1.0);
```
Checking for `runtime == null` would keep it working for local testing (unless you add an explicit `runtime.maxVersion` to the test interface).

Reply:
Looking forward to optional chaining in ES2020 ...

Could refactor this to TypeScript to use it here.